src/com/epam/javatrainings/classdecomposition/model/Pizza.java

```diff
@@ -0,0 +1,127 @@
+package com.epam.javatrainings.classdecomposition.model;
+
+import com.epam.javatrainings.classdecomposition.util.PizzaUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Hayk on 31.08.2021.
+ */
+public class Pizza {
+
+    private final double basePrice = 1;
+    private double price;
+    private int quantity;
+    private int orderNumber;
+    private String name;
+    private List<Ingredient> ingredients;
+    private PizzaType type;
+
+    public Pizza(String name, List<Ingredient> ingredients) {
+        this.name = name;
+        this.ingredients = ingredients;
+        this.type = PizzaType.REGULAR;
+    }
+
+    public Pizza(double price, int quantity, int orderNumber, String name, PizzaType type) {
```

Comment:
if you have 5 different constructor, its the right time to think about builder pattern